internal/names.go

```diff
@@ -24,9 +24,13 @@ const (
 
 	// Namespace is the reverse DNS namespace used with labels and CNAB custom actions.
 	Namespace = "com.docker.app."
+	// CnabNamespace is the namespace used with the CNAB well known custom actions
+	CnabNamespace = "io.cnab."
 
-	// ActionStatusName is the name of the custom "status" action
-	ActionStatusName = Namespace + "status"
+	// ActionDeprecatedStatusName is the name of the deprecated custom "status" action
```

Comment:
Might want to add `Deprecated: use ActionStatusName instead` to make IDE's mark it deprecated

Reply:
Good point. Added